tox.ini

```diff
@@ -10,8 +10,20 @@ envlist = bandit, safety, black, black-check, isort, isort-check, fix-copyright,
 ; when running locally we don't want to fail for no good reason
 skip_missing_interpreters = true
 
+[packages-deps]
+deps = 
+    gym==0.15.6
+    aiohttp==3.7.4.post0
+    gym==0.15.6
+    hypothesis==6.21.6
+    numpy>=1.18.1
+    openapi-core==0.13.2
+    openapi-spec-validator==0.2.8
+    asn1crypto==1.4.0
```

Comment:
this is a plugin dep?

Reply:
`gym` is listed twice

Reply:
It's also a dep for the p2p connection package https://github.com/valory-xyz/open-aea/commit/78be3122f64664030151d8110911c9ac5a5ffd32

Reply:
@angrybayblade what about `gym` listed twice? Remove in future PR